packages/react-component-library/cypress/specs/timeline/index.spec.ts

```diff
@@ -264,6 +264,18 @@ describe('Compound Timeline', () => {
         it('should fill the width', () => {
           cy.get(selectors.timeline.month).should('have.css', 'width', '1023px')
         })
+
+        describe('resize window', () => {
```

Comment:
Would be good to make this consistent with the other `describe`s – perhaps something like:

```suggestion
        describe('resizing the window', () => {
```